tasks/main.yml

```diff
@@ -80,7 +78,7 @@
         group={{ project_user }}
         mode=600
   register: github_key
-  when: github_deploy_key is defined
+  when: github_deploy_key|default('')
 
 # TODO: come up with a safer way of exposing the local repo
```

Comment:
Isn't this commented-out rsync code implemented down below for task "sync source from local directory"?  IOW, it looks to me that this commented out stuff starting with "# TODO" can be removed.